### DISCUSSION TITLE
有用有办法直接安装到Quest3上？

### DISCUSSION THREAD

**Discussion by reehon** [AlistGo/alist#6361] [Ideas]
目前了解的关于WebDAV的方式好像都是通过局域网里的电脑配置好Alist然后VR头显设备访问；
有没有办法直接安装到Quest3里，这样即使在户外，连上手机热点就可以直接看网盘里的电影了...郊游的时候，躺椅上的户外影院~

**Comment by GD2021** [comment]
有安卓版

**Reply by ArcherKev** [reply]
quest3应该选择哪一个版本呢？
